redhat-director-scripts/docker/build_containers.sh

```diff
@@ -34,22 +34,67 @@ cp -R $base_dir/trilio-horizon-plugin $base_dir/${build_dir}/
 
 echo -e "Creating trilio-datamover container for rhosp13"
 cd $base_dir/${build_dir}/trilio-datamover/
-docker build --no-cache -t trilio/trilio-datamover:${tvault_version} .
+docker build --no-cache -t trilio/trilio-datamover:${tvault_version}-rhosp13 .
 
 #Build trilio_datamover-api containers
 
 echo -e "Creating trilio-datamover container-api for rhosp13"
 cd $base_dir/${build_dir}/trilio-datamover-api/
-docker build --no-cache -t trilio/trilio-datamover-api:${tvault_version} .
+docker build --no-cache -t trilio/trilio-datamover-api:${tvault_version}-rhosp13 .
 
 
 ## Build horizon plugin
 
 echo -e "Creating trilio horizon plugin container for rhosp13"
 cd $base_dir/${build_dir}/trilio-horizon-plugin/
-docker build --no-cache -t trilio/trilio-horizon-plugin:${tvault_version} .
+docker build --no-cache -t trilio/trilio-horizon-plugin:${tvault_version}-rhosp13 .
 
 
 
+# Clean the build_dir
+rm -rf $base_dir/${build_dir}
+
+#################################################################################################
+
+################## Build RHOSP15 containers ####################
+
+## Work dir preparation for for rhosp15
+build_dir=tmp_docker_rhosp15_${tvault_version}
+rm -rf $base_dir/${build_dir}
+mkdir -p $base_dir/${build_dir}
+
+cp -R $base_dir/trilio-datamover $base_dir/${build_dir}/
+cp -R $base_dir/trilio-datamover-api $base_dir/${build_dir}/
+cp -R $base_dir/trilio-horizon-plugin $base_dir/${build_dir}/
+
+#Build trilio-datamover containers for rhosp15
+
+echo -e "Creating trilio-datamover container for rhosp15"
+cd $base_dir/${build_dir}/trilio-datamover/
+rm Dockerfile
+cp Dockerfile_rhosp15 Dockerfile
+buildah bud -t docker.io/trilio/trilio-datamover:${tvault_version}-rhosp15 .
+podman push docker.io/trilio/trilio-datamover:${tvault_version}-rhosp15
+
+
+
+#Build trilio_datamover-api containers for rhosp15
+
+echo -e "Creating trilio-datamover container-api for rhosp13"
```

Comment:
Please change echo message to "creating trilio-datamover container-api for rhosp15" instead of 'rhosp13. at every occurance in file

Reply:
Added the changes